rnndb/falcon.xml

```diff
@@ -412,6 +417,25 @@
 	</reg32>
 	<!-- XXX: GM107 PDAEMON has something at 244+ -->
 
+	<reg32 offset="0xb00" name="CURRENT_CTX" variants="GM107-">
+		<bitfield low="0"  high="27" name="PTR"/>
```

Comment:
Merge PTR and TARGET into one bitfield, of type g80_channel -- avoids duplicating the subfields.